src/Microsoft.Health.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -550,6 +569,33 @@ public override void Set(SqlDataRecord record, int ordinal, byte? value)
         }
     }
 
+    public class NullableUniqueIdentifierColumn : Column<Guid?>
```

Comment:
Quick question: does this also mean only 1 row can have the 'null' identifier. Does this make sense?

Reply:
I'm sorry, could you rephrase your question? Like the other nullable column types in this file, I'm pretty sure they refer to a representation of some table column type that has been annotated as nullable.

For example, given the SQL table definition:
```SQL
CREATE TABLE Foo
(
    -- ...
    Bar INT NULL,
    -- ...
)
```

In C# we'll code-gen the corresponding field for its table type:
```csharp
internal readonly NullableIntColumn Bar = new NullableIntColumn("Bar");
```

But multiple columns can be annotated as "nullable" for a given row, and multiple rows can contain `NULL` values in these "nullable" columns.

Reply:
Initially I was thinking there was also a uniqueness constraint here, where in re-reading, this change for `UniqueIdentifier` is just the datatype. Sorry for the confusion :)

Reply:
Ah no problem! Yeah -- it seems that we're missing support for the type `uniqueidentifier`, so this PR is adding it